torch/csrc/utils/tensor_numpy.cpp

```diff
@@ -125,20 +125,20 @@ PyObject* tensor_to_numpy(const at::Tensor& tensor) {
   return array.release();
 }
 
-at::Tensor tensor_from_numpy(PyObject* obj, bool warn_if_not_writeable/*=true*/) {
+at::Tensor tensor_from_numpy(PyObject* obj, bool warn_if_not_writable/*=true*/) {
   if (!PyArray_Check(obj)) {
     throw TypeError("expected np.ndarray (got %s)", Py_TYPE(obj)->tp_name);
   }
   auto array = (PyArrayObject*)obj;
 
   // warn_if_not_writable is true when a copy of numpy variable is created.
   // the warning is suppressed when a copy is being created.
-  if (!PyArray_ISWRITEABLE(array) && warn_if_not_writeable) {
+  if (!PyArray_ISWRITABLE(array) && warn_if_not_writable) {
```

Comment:
PyArray_ISWRITEABLE must stay as it was because it's a function call, and the function call uses an incorrect spelling of "writable" but we can't change it.

Reply:
Okay, I have changed that.